pkg/util/crd_defaults.go

```diff
@@ -43,7 +43,7 @@ func GetBlackDuckTemplate() *blackduckv1.BlackduckSpec {
 		DbPrototype:       "",
 		CertificateName:   "default",
 		Type:              "",
-		Version:           "2019.4.2",
+		Version:           "",
```

Comment:
Probably we can remove all the empty fields from the struct. Default value for String is "".